pkg/process/checks/process_events_common.go

```diff
@@ -18,3 +22,51 @@ func parseRFC3339Time(t *testing.T, s string) time.Time {
 	require.NoError(t, err)
 	return parsed
 }
+
+// FmtProcessEvents formats process lifecyle events to be sent in an agent payload
+func FmtProcessEvents(events []*model.ProcessEvent) []*payload.ProcessEvent {
```

Comment:
Since `cmd/process-agent/app/events.go` doesn't use build tag, this function needs to be available in all platforms and it has been moved to the `_common` file.